src/columns/fast-formula-parser.ts

```diff
@@ -0,0 +1,85 @@
+import * as glide from "../glide";
+
+import FormulaParser from 'fast-formula-parser';
+
+const run = async (formula, ...params) => {
+    var data: any[][] = [];
+
+    const parser = new FormulaParser({
+        onCell: ({_, row, col}) => {
+            return data[row - 1][col - 1];
+        },
+        onRange: (ref: { from: { row: any; col: any; }; to: { row: number; col: number; }; }) => {
```

Comment:
Why are the `row` and `col` in `from` typed `any` but the ones in `to` are `number`?